docs/feature-store/transformations.md

```diff
@@ -241,6 +241,63 @@ config = fstore.RunConfig(local=False, function=my_func, handler="ingest_handler
 fstore.ingest(feature_set, source, run_config=config, spark_context=spark_service_name)
 ```
 
+### Spark operator ingestion example
+When using spark operator execution the MLRun run execution details would be returned, allowing tracking of its status and results.
```

Comment:
How about:
When running with spark operator, the MLRun execution details are returned, allowing tracking of the job's status and results.